projects/daq2/zcu102/system_bd.tcl

```diff
@@ -1,15 +1,16 @@
+## Offload attributes
+set adc_offload_type 0
+set adc_offload_size [expr 960 * 1024]
```

Comment:
On my machine I was able to build with 1024 * 1024, both ADC/DAC FIFOs used 258 BRAM Tiles out of the 912 available